### PR TITLE
Add absent math constants

### PR DESCRIPTION
**Pull request by drdzyk**
Add absent math constants from https://www.php.net/manual/en/math.constants.php

### CODE REVIEW

File: tests/phpt/dl/1029_inf.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@ok
+<?php
+
+function test_inf() {
+  $values = [exp(800), pow(0, -2), INF];
```
Comment (AlexK0) [inline tests/phpt/dl/1029_inf.php:5]: nit: pls add:
-INF 
INF + INF
INF * INF
INF - INF
INF / INF

Comment (drdzyk) [reply tests/phpt/dl/1029_inf.php]: done

**Review by quasilyte — APPROVED** [review]

**Review by AlexK0 — APPROVED** [review]

File: functions.txt [hunk 2]
```diff
@@ -516,6 +516,22 @@ define('PHP_URL_FRAGMENT', 7);
...
+define('M_PI_4', 0.78539816339744830962);
+define('M_1_PI', 0.31830988618379067154);
+define('M_2_PI', 0.63661977236758134308);
+define('M_SQRTPI', 1.77245385090551602729);
+define('M_2_SQRTPI', 1.12837916709551257390);
+define('M_SQRT2', 1.41421356237309504880);
+define('M_SQRT3', 1.73205080756887729352);
+define('M_SQRT1_2', 0.70710678118654752440);
+define('M_LNPI', 1.14472988584940017414);
+define('M_EULER', 0.57721566490153286061);
```
Comment (unserialize) [inline functions.txt:535]: Please, add the `is_infinite()` function (it exists in PHP, like `is_nan()`)

Comment (drdzyk) [reply functions.txt:535]: This function is already implemented here: functions.txt:307. Also it used in new test tests/phpt/dl/1029_inf.php.

**Review by unserialize — APPROVED** [review]

**Pull request merged**